### ISSUE TITLE
Feat: Change the default title of news

### ISSUE THREAD

**Issue by Relf-Algoo** [tracim/tracim#5520]
## Feature description and goals

The default title for news is "News of" + username + date and time. For example : "News of Charline on 3/7/2022, 11:36:03 AM"
We think it would be better if the date and time were at the beginning of the title, with the date in a YYYYMMDD format (same format for each and every language). In the tilte of news, the date is the most important part and should be at the beginning to be seen more easily. It also would help with sorting the news (when the sorting will be available, for example in the search results.)

**Comment by Relf-Algoo** [comment]
Meh.
First parts of messages are often greetings or similar. Mostly they are not significant.

**Comment by Haskaris** [comment]
> First parts of messages are often greetings or similar. 

I disagree on that, my current usage of news is to publish stuff on Tracim (seems fair to me). Sometimes it's indeed used as greetings, but sometimes it's used as a thread of discussion. Therefore I change the title to, for example "Help: how can I do..."
